crypto_kem/kyber768/clean/poly.c

```diff
@@ -20,13 +20,10 @@ void PQCLEAN_KYBER768_poly_compress(unsigned char *r, const poly *a) {
 
     for (i = 0; i < KYBER_N; i += 8) {
         for (j = 0; j < 8; j++) {
-            t[j] = (((PQCLEAN_KYBER768_freeze(a->coeffs[i + j]) << 3) +
-                     KYBER_Q / 2) /
-                    KYBER_Q) &
-                   7;
+            t[j] = (((PQCLEAN_KYBER768_freeze(a->coeffs[i + j]) << 3) + KYBER_Q / 2) / KYBER_Q) & 7;
         }
 
-        r[k] = t[0] | (t[1] << 3) | (t[2] << 6);
+        r[k]     = t[0]        | (t[1] << 3) | (t[2] << 6);
```

Comment:
Did astyle do all these math alignments itself, or did you do them manually and it was willing to preserve them?

Reply:
The latter.